plugin/storage/cassandra/dependencystore/storage_test.go

```diff
@@ -268,5 +268,5 @@ func TestGetBuckets(t *testing.T) {
 }
 
 func matchEverything() any {
-	return mock.MatchedBy(func(v []any) bool { return true })
+	return mock.MatchedBy(func(_ /* v */ []any) bool { return true })
```

Comment:
`v` unnecessary

Reply:
and `_`

Reply:
I thought the `_` would be better for readability based on last times PR ?

Reply:
`_` is ok-ish